src/index.js

```diff
@@ -40,6 +42,7 @@ module.exports = {
         ...fullCorvidTypes.BASE,
         ...fullCorvidTypes.BACKEND,
         ...getJswTypeDeclarations(),
+        ...getJsonTypeDeclarations(),
         ...getNpmDependenciesTypesDeclarations(dependencies)
```

Comment:
why aren't these 2 part of the base or some other common thingy ?

Reply:
done